packages/medusa/src/api-v2/admin/products/route.ts

```diff
@@ -22,7 +22,8 @@ export const GET = async (
       skip: req.listConfig.skip,
       take: req.listConfig.take,
     },
-    fields: defaultAdminProductFields,
+    // TODO: How do we include the relations as well as defined in query-config? Or will remote query work differently?
```

Comment:
@adrien2p let's say I want to get the variants relationship from the FE, how is that expressed with remoteQuery?

Reply:
you can just add it in the remote query, the transform to remote query object should do it